### ISSUE TITLE
WSDL Generation Fails with 500 Server Error

### ISSUE THREAD

**Issue by mam08ixo** [dotmailer/dotmailer-magento2-extension#560]
Environment:
* Magento 2.3.5
* Dotdigitalgroup_Email 4.4.0

Steps to Reproduce:
* Access WSDL, e.g. https://base.url/soap/default?wsdl&services=quoteGuestCartManagementV1

Expected Result:
* WSDL schema loads

Actual Result:
* `env:Receiver` fault with reason text 

> The "\DateTime" parameter type is invalid. Verify the parameter and try again.

**Comment by mam08ixo** [comment]
Seems to be fixed with #557 but not yet included with M2.

**Comment by sta1r** [comment]
Hi @mam08ixo, indeed this was fixed quite recently and won't be included in a Magento release until 2.4. In the meantime it's possible to upgrade our connector via:
```
composer require dotmailer/dotmailer-magento2-extension:"4.5.1 as 4.4.0"
```

**Comment by umarch06** [comment]
@sta1r this is a very critical issue as none is able to get into swagger UI. Is there any Magento patch available for this?

**Comment by sta1r** [comment]
@umarch06 is the upgrade mentioned above not possible?

**Comment by YiffyToys** [comment]
> 
> 
> Hi @mam08ixo, indeed this was fixed quite recently and won't be included in a Magento release until 2.4. In the meantime it's possible to upgrade our connector via:
> 
> ```
> composer require dotmailer/dotmailer-magento2-extension:"4.5.1 as 4.4.0"
> ```

Thanks a lot for this.
However the syntax seems to be off.
I get:

" Could not find package as."

**Comment by rsjmeijer** [comment]
For us upgrading dotmailer worked fine!
